samples/csharp/common/AutoML/ConsoleHelper.cs

```diff
@@ -89,6 +98,11 @@ internal static void PrintIterationMetrics(int iteration, string trainerName, Re
             CreateRow($"{iteration,-4} {trainerName,-35} {metrics?.RSquared ?? double.NaN,8:F4} {metrics?.MeanAbsoluteError ?? double.NaN,13:F2} {metrics?.MeanSquaredError ?? double.NaN,12:F2} {metrics?.RootMeanSquaredError ?? double.NaN,8:F2} {runtimeInSeconds.Value,9:F1}", Width);
         }
 
+        internal static void PrintIterationMetrics(int iteration, string trainerName, RankingMetrics metrics, double? runtimeInSeconds)
+        {
+            CreateRow($"{iteration,-4} {trainerName,-9} {metrics?.NormalizedDiscountedCumulativeGains[0] ?? double.NaN,9:F4}, {metrics?.NormalizedDiscountedCumulativeGains[1] ?? double.NaN,9:F4}, {metrics?.NormalizedDiscountedCumulativeGains[2] ?? double.NaN,9:F4} {metrics?.DiscountedCumulativeGains[0] ?? double.NaN,9:F4} {runtimeInSeconds.Value,9:F1}", Width);
```

Comment:
Printing NDCG@1,3,10 & DCG@10
```suggestion
            CreateRow($"{iteration,-4} {trainerName,-9} {metrics?.NormalizedDiscountedCumulativeGains[0] ?? double.NaN,9:F4}, {metrics?.NormalizedDiscountedCumulativeGains[2] ?? double.NaN,9:F4}, {metrics?.NormalizedDiscountedCumulativeGains[9] ?? double.NaN,9:F4} {metrics?.DiscountedCumulativeGains[9] ?? double.NaN,9:F4} {runtimeInSeconds.Value,9:F1}", Width);
```

Reply:
`NormalizedDiscountedCumulativeGains` and `DiscountedCumulativeGains` only has a length of 3 in the iterations until we get the option to pass in how many of these we want. Should we keep it printing indexes 0, 1, 2, for nDCG and 2 for DCG?

Reply:
Interesting. Where does the limit come from?

Reply:
Looks like the limit come from here:
https://github.com/dotnet/machinelearning/blob/f87a3bbd8adb12934dae0a0060813ce9b7500664/src/Microsoft.ML.Data/Evaluators/RankingEvaluator.cs#L34-L38
```c#
        /// <value>
        /// Maximum truncation level for computing (N)DCG
        /// </value>
        [Argument(ArgumentType.AtMostOnce, HelpText = "Maximum truncation level for computing (N)DCG", ShortName = "t")]
        public int DcgTruncationLevel = 3;
```

Few choices:
* Print NDCG/DCG@1,2,3 (as you purpose)
* Change the parameter [default](https://github.com/dotnet/machinelearning/blob/f87a3bbd8adb12934dae0a0060813ce9b7500664/src/Microsoft.ML.Data/Evaluators/RankingEvaluator.cs#L38) from 3 to 10. Should be safe as it simply returns a longer array of DCG/NDCG values.
* Hard code a value to 10 in the AutoML ranking code -- set `var rankingEvaluatorOptions = new RankingEvaluatorOptions { DcgTruncationLevel = 10 };` at https://github.com/dotnet/machinelearning/blob/f87a3bbd8adb12934dae0a0060813ce9b7500664/src/Microsoft.ML.AutoML/Experiment/MetricsAgents/RankingMetricsAgent.cs#L62
* Larger PR to expose the `DcgTruncationLevel` to the AutoML API (as mentioned above: https://github.com/dotnet/machinelearning-samples/pull/852#discussion_r501628536)

Reply:
I just noticed that's where it comes from, as well. 😄

If this isn't in a hurry to get merged in I could make a PR to set the default to 10 and then update this once it gets released. I could also do the reverse to print 1,2, and 3 for now, put the PR in to default to 10, then update this once it gets released.

I'll also create the issue to expose the truncation level to the AutoML API to keep track of the longer term fix.

Reply:
Issue created [here](https://github.com/dotnet/machinelearning/issues/5425) to implement the `DcgTruncationLevel` into the AutoML API.